modules/osdk-updating-128-to-131.adoc

```diff
@@ -0,0 +1,243 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-updating-projects.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-helm-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc
+// * operators/operator_sdk/java/osdk-java-updating-projects.adoc
+
+ifeval::["{context}" == "osdk-golang-updating-projects"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-updating-projects"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-updating-projects"]
+:helm:
+:type: Helm
+endif::[]
+ifeval::["{context}" == "osdk-hybrid-helm-updating-projects"]
+:hybrid:
+:type: Hybrid Helm
+endif::[]
+ifeval::["{context}" == "osdk-java-updating-projects"]
+:java:
+:type: Java
+endif::[]
+
+:_content-type: PROCEDURE
+[id="osdk-upgrading-projects_{context}"]
+= Updating {type}-based Operator projects for Operator SDK {osdk_ver}
+
+The following procedure updates an existing {type}-based Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+* Operator SDK {osdk_ver} installed
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+ifdef::golang,hybrid,java[]
+* Edit your Operator project's makefile to update the Operator SDK version to {osdk_ver} as shown in the following example:
++
+.Example makefile
+[source,make]
+----
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
```

Comment:
In the upstream SDK project migration docs, the migrations for the 1.31.0 release all use 1.30.0 for the version. This applies to `OPERATOR_SDK_VERSION` in the makefiles, and for the versions of the Helm and Ansible Operators. 

Should this doc also use 1.30.0? Or is it safe to assume that 1.31.0 is correct?

Reply:
1.31.0 should be correct. I'm assuming there was a step missed somewhere that resulted in it still being v1.30.0, but it should be 1.31.0